packages/api/tests/dwn-api.spec.ts

```diff
@@ -569,4 +571,57 @@ describe('DwnApi', () => {
       });
     });
   });
+
+  describe('permissions.grant()', () => {
```

Comment:
Could a test for sending a `PermissionsGrant` to a remote DWN be added in this PR?

Reply:
If I understand agents correctly, setting `dwn.permissions.grant(target  : bobDid.did, ...)` will send to a remote. We do that in the test in `describe('target: did'`